templates/coderd.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
+    app: {{ include "coder.serviceName" . }}
```

Comment:
ah, I wonder if it's related to https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
hmm maybe we should set some of these as well

Reply:
I think Helm adds a bunch. Not a big deal, can always do it later!